### PR TITLE
Fixed test cases and policies.  Re-based the opa library

### PR DESCRIPTION
**Pull request by mangchiandjjoe**

### CODE REVIEW

**Comment by mangchiandjjoe** [comment]
@douglas-reid could you please take a look and add lgtm label?

File: mixer/adapter/opa/opa_integration_test.go [hunk 1]
```diff
@@ -242,18 +241,18 @@ func xTestServer(t *testing.T) {
...
 		},
 		"Denied request": {
 			attrs: map[string]interface{}{
 				"destination.service": "svc.cluster.local",
 				"source.uid":          "janet",
-				"request.path":        "/detail/janet",
-				"target.service":      "invalid",
-				"source.service":      "landing_page",
+				"request.path":        "/detail/ken",
+				"target.service":      "landing_pages",
```
Comment (douglas-reid) [inline mixer/adapter/opa/opa_integration_test.go]: i don't want to block this PR, but what is the difference between `target.service` and `destination.service` and how do these relate to attributes?

Comment (mangchiandjjoe) [reply mixer/adapter/opa/opa_integration_test.go]: destination.service is not used in the test policy. It can be removed.

Comment (mangchiandjjoe) [reply mixer/adapter/opa/opa_integration_test.go]: I was wrong. destination.service  is required.
target service is used to fill input.action.service in the test configuration.

**Pull request merged**